src/presence_analyzer/tests.py

```diff
@@ -133,20 +133,53 @@ def test_start_end_view(self):
             value['start'] = value['start'].strftime('%H:%M:%S')
             value['end'] = value['end'].strftime('%H:%M:%S')
 
-        response = self.client.get('/api/v1/presence_start_end/10')
+        response = self.client.get('/api/v1/presence_start_end_weekday/10')
         data = json.loads(response.data)
 
         self.assertEqual(response.status_code, 200)
         self.assertEqual(expected_data, data)
 
-    def test_start_end_view_for_non_existent_user(self):
+    def test_start_end_weekday_view_for_non_existent_user(self):
         """
-        Check response for user with id 1 not mentioned in test_data.csv is 404
+        Check response for user with id 1 not mentioned in test_data.csv is 404.
         """
         response = self.client.get('/api/v1/presence_start_end/1')
 
         self.assertEqual(response.status_code, 404)
 
+    def test_mean_time_month_view(self):
+        """
+        Test JSON response for user 10.
+        """
+        response = self.client.get('/api/v1/mean_time_month/10')
+
+        received_data = json.loads(response.data)
+        months = utils.group_intervals_by_month(utils.get_data()[10])
+        # +1 because  month_abbr is shifted by 1 to match January with index 1.
+        expected_data = [
+            [calendar.month_abbr[month+1], intervals]
+            for month, intervals in enumerate(months)
+        ]
+
+        for data in expected_data:
+            average_time = utils.convert_seconds_to_time(int(utils.mean(data[1])))
+            data[1] = [
+                average_time.hour,
+                average_time.minute,
+                average_time.second
+            ]
```

Comment:
Don't calculate it, in tests u should compare results from api with static data that u expect.